tests/app/test_draft.py

```diff
@@ -249,16 +249,16 @@ def test_draft_modified_since_last_read_true(self):
 
         with self.app.app_context():
             with current_app.test_request_context():
-                is_valid_draft = Retriever().check_msg_id_is_a_draft(msg_id, self.user_respondent)
-        self.assertTrue(is_valid_draft[0])
+                draft = Retriever().get_draft(msg_id, self.user_respondent)
```

Comment:
Are these static methods on the Retriever class? If so, we can get rid of the `()`.